test/views/templates/PaymentsHistoryChargeHelperSpec.scala

```diff
@@ -30,6 +30,45 @@ class PaymentsHistoryChargeHelperSpec extends UnitSpec {
           result shouldBe Some(paymentHistoryChargeType)
         }
       }
+
+      "return a Vat Inaccuracy Assessments Pen charge type" in {
```

Comment:
Are these not tested by the above test?
Is it mandatory that we use hard coded strings, because the ChargeTypes were not tested with hard coded strings, they were done similarly to above (which is why I started automating the test).

Reply:
Completely agree. Forgot to take them out in the rebase